src/tests/users/signup.test.js

```diff
@@ -1,30 +1,268 @@
+      query: `
+        mutation {
+          verifyEmail (verificationCode: "${users[0].verificationCode}") {
+            record 
+            error {
+              message
+            }
+          }
+        }
+      `
+    })
+      .expect((res) => {
+        expect(res.body).toHaveProperty('data.verifyEmail.error.message')
+        expect(res.body.data.verifyEmail.error.message).toStrictEqual(
+          'user verified already'
+        )
+      })
+      .expect(200)
+  })
+
+  it('there are more than 0 mentors in database', () => {
+    return request({
+      query: `
+        query {
+          mentors {
+            record {
+              username
+              email
+            }
+          }
+        }
+      `
+    })
+      .expect((res) => {
+        expect(res.body).toHaveProperty('data.mentors.record')
+        expect(res.body.data.mentors.record.length).toBeGreaterThan(0)
+      })
+      .expect(200)
+  })
 
   it('should create a new user', () => {
     return request({
       query: `
         mutation {
           signup(record: {
-              fullName: "${fakeUser.fullName}",
-              email: "${fakeUser.email}",
-              password: "${fakeUser.password}"
+              firstName: "${fakeUser2.firstName}"
+              lastName: "${fakeUser2.lastName}"
+              email: "${fakeUser2.email}"
+              password: "${fakeUser2.password}"  
+              username: "${fakeUser2.username}"
+          }) {
+            record {
+              _id
+              email
+            }
+          }
+        }
+      `
+    })
+      .expect((res) => {
+        expect(res.body).toHaveProperty('data.signup.record')
+        expect(res.body.data.signup.record.email).toStrictEqual(
+          fakeUser2.email
+        )
+      })
+      .expect(200)
+  })
+
+  it('should create a new user with mentor and atleast 5 skills and 5 knowledge', () => {
+    return request({
+      query: `
+        mutation {
+          signup(record: {
+              firstName: "${fakeUser.firstName}"
+              lastName: "${fakeUser.lastName}"
+              email: "${fakeUser.email}"
+              password: "${fakeUser.password}"  
+              username: "${fakeUser.username}"
+              mentor: "${users[0]._id}"
+              skills: [${skills}]
+              knowledge: [${knowledge}]
           }) {
             record {
-              fullName
+              _id
               email
+              skills { title }
+              knowledge { title }
             }
           }
         }
       `
     })
-      .expect(res => {
+      .expect((res) => {
         expect(res.body).toHaveProperty('data.signup.record')
         expect(res.body.data.signup.record.email).toStrictEqual(fakeUser.email)
+        expect(
+          res.body.data.signup.record.skills.length
+        ).toBeGreaterThanOrEqual(5)
+        expect(
+          res.body.data.signup.record.knowledge.length
+        ).toBeGreaterThanOrEqual(5)
```

Comment:
```suggestion
        expect(
          res.body.data.signup.record.knowledge.length
        ).toStrictEqual(knowledge.length)
```

Reply:
this is testing valid number of skills in database for signup to be successful